src/errors.rs

```diff
@@ -95,6 +106,16 @@ impl Display for Error {
             }
             Error::InvalidSignature => write!(f, "Failed signature validation"),
             Error::NetworkOther(ref error) => write!(f, "Error on Vault network: {}", error),
+            Error::LossOfPrecision => {
+                write!(f, "Lost precision on the number of coins during conversion")
```

Comment:
Should be "parsing" rather than `conversion".  Same comment for `description()` variant.